eclair-core/src/test/scala/fr/acinq/eclair/wire/protocol/LightningMessageCodecsSpec.scala

```diff
@@ -345,7 +347,7 @@ class LightningMessageCodecsSpec extends AnyFunSuite {
     val accept = AcceptChannel(randomBytes32(), 3 sat, UInt64(4), 5 sat, 6 msat, 7, CltvExpiryDelta(8), 9, publicKey(1), point(2), point(3), point(4), point(5), point(6))
     val funding_created = FundingCreated(randomBytes32(), bin32(0), 3, randomBytes64())
     val funding_signed = FundingSigned(randomBytes32(), randomBytes64())
-    val funding_locked = FundingLocked(randomBytes32(), point(2))
+    val channel_ready = ChannelReady(randomBytes32(), point(2))
```

Comment:
Can you add one where `channel_ready` contains an `alias`?

Reply:
Done in fc7d4ee88, but this test needs serious refactoring:
- test cases should be defined in a collection, not one-by-one
- should do non-reg with expected binary data instead of just roundtrip